### DISCUSSION TITLE
Strange error

### DISCUSSION THREAD

**Discussion by Tonaion02** [bevyengine/bevy#14535] [Q&A]
I am only using DefaultPlugins and i am getting this error:
![image](https://github.com/user-attachments/assets/09ac6e66-f1e9-4ca8-a782-59a027319942)
Someone can figure why?

**Comment by hymm** [comment]
What features do you have enabled? Something might not be `cfg`ed out correctly in bevy.

**Reply by Tonaion02** [reply]
The enabled features are the following:
![image](https://github.com/user-attachments/assets/1d184dab-e327-45b8-a123-b4941b875615)
I don't know if can help, but this error happened only when i don't run directly with "run" method but i use the update function in a loop.
